cms/envs/common.py

```diff
@@ -732,6 +732,16 @@
             continue
     INSTALLED_APPS += (app_name,)
 
+############## Milestones ##############
+# Turn off milestones app by default
+FEATURES['MILESTONES_APP'] = False
```

Comment:
The milestones app installation should instead be handled with the OPTIONAL_APPS functionality found above on line  706